### PR TITLE
Add variables for configure

### PR DESCRIPTION
**Pull request by Shougo**
I have added configurable variables for deoplete-tabnine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,12 @@ Plug 'tbodt/deoplete-tabnine', { 'do': 'powershell.exe .\install.ps1' }
...
   [Deoplete]: https://github.com/Shougo/deoplete.nvim/
   [TabNine]: https://tabnine.com
+
+Configuration:
+
+```vim
+call deoplete#custom#var('tabnine', {
+\ 'line_limit': 500,
+\ 'max_num_results': 20,
+\ })
```
Comment (tbodt) [inline README.md:40]: Could you document what the options do? line_limit isn't a super obvious name.

**Review by tbodt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Shougo) [reply README.md:40]: OK.  I will add the documentation.

Comment (Shougo) [reply README.md:40]: I have added it.

Comment (tbodt) [reply README.md:40]: Can you format it so it displays nicely in markdown?

Comment (Shougo) [reply README.md:40]: I have improved the documentation.

Comment (tbodt) [reply README.md:40]: Thanks!

**Comment by Shougo** [comment]
Thanks!

**Pull request merged**